imports/plugins/core/email/server/no-meteor/util/getShopLogo.js

```diff
@@ -8,6 +6,9 @@ import { Media } from "/imports/plugins/core/files/server";
  * @return {String} Absolute image URL
  */
 export default async function getShopLogo(context, shop) {
+  const { collections } = context;
```

Comment:
Could these be combined into one line? I don't see us using `collections` anywhere else

Reply:
Yes I do this by habit since we have other places where we need it.